django/thunderstore/api/urls.py

```diff
@@ -110,4 +111,4 @@
         TeamServiceAccountListAPIView.as_view(),
         name="cyberstorm.team.service-account",
     ),
-]
+] + plugin_registry.get_package_api_urls()
```

Comment:
Given that we're appending these to the cyberstorm API url namespace, I feel we should make the plugin function reflect that and call it e.g. `get_cyberstorm_api_urls` or such. At least calling it `get_package_api_urls` doesn't make a lot of sense as it could actually provide any arbitrary API urls, not _just_ package API urls